### PR TITLE
Magic bracelets

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Mods "Adding magic bracelets"

#### Purpose of change

adding some cool magic bracelets, mostly oriented to save your life in critical situation

#### Describe the solution

4 new bracelets:

missile volley bracelet - activated a bunch of mana bolts to damage enemies nearby
petrification bracelet - totally freeze your enemies in big range
disorientation bracelet - disorient your enemies in smaller range
propulsion bracelet - make you really fast

#### Describe alternatives you've considered

transform bracelets to bracers

#### Testing

works fine

### CODE REVIEW

File: data/mods/Magiclysm/items/enchanted_bracelet.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    "sided": true,
+    "flags": [ "FANCY", "BELTED", "NO_WEAR_EFFECT" ],
+    "armor": [ { "coverage": 50, "covers": [ "hand_l", "hand_r" ], "specifically_covers": [ "hand_wrist_l", "hand_wrist_r" ] } ]
+  },
+  {
+    "copy-from": "platinum_bracelet_magic",
+    "id": "missile_volley_bracelet",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "missile volley bracelet" },
+    "description": "A bracelet, completely covered with a large amount of runes from both sides.  Wizards love to name it \"ultima ratio\", describing his colossal might.  Require almost a whole week to restore his charge.",
```
Comment (TheShadowFerret) [inline data/mods/Magiclysm/items/enchanted_bracelet.json:21]:
```suggestion
    "description": "A bracelet, completely covered with a large amount of runes from both sides.  Wizards love to name it \"Ultima Ratio\", describing its colossal might.  Require almost a whole week to restore its charge.",
```
I don't think rings have a sex 🤔

File: data/mods/Magiclysm/items/enchanted_bracelet.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "holster": true, "ammo_restriction": { "crystallized_mana": 1 } } ],
+    "relic_data": { "charge_info": { "recharge_type": "periodic", "time": "7 d", "regenerate_ammo": true } },
+    "use_action": { "type": "cast_spell", "spell_id": "missile_volley", "no_fail": true, "level": 0, "need_worn": true }
+  },
+  {
+    "copy-from": "platinum_bracelet_magic",
+    "id": "gorgon_bracelet",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "petrification bracelet" },
+    "description": "A bracelet, completely covered with a large amount of runes from both sides.  You don't really know how it works, but after using enemies literally freeze in the middle of their movement.  Require almost 7 days to restore his charge.",
```
Comment (TheShadowFerret) [inline data/mods/Magiclysm/items/enchanted_bracelet.json:44]:
```suggestion
    "description": "A bracelet, completely covered with a large amount of runes from both sides.  You don't really know how it works, but after use enemies literally freeze in the middle of their movement.  Require almost 7 days to restore its charge.",
```

File: data/mods/Magiclysm/items/enchanted_bracelet.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "holster": true, "ammo_restriction": { "crystallized_mana": 1 } } ],
+    "relic_data": { "charge_info": { "recharge_type": "periodic", "time": "7 d", "regenerate_ammo": true } },
+    "use_action": { "type": "cast_spell", "spell_id": "gorgon_spell", "no_fail": true, "level": 0, "need_worn": true }
+  },
+  {
+    "copy-from": "platinum_bracelet_magic",
+    "id": "disorientation_bracelet",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "disorientation bracelet" },
+    "description": "Known as a \"worse version of petrification bracelet\", this tool cast a spell, that temporary break the whole movement system of creature, disabling his movement for a short time.  Require almost week to restore his charge.",
```
Comment (TheShadowFerret) [inline data/mods/Magiclysm/items/enchanted_bracelet.json:67]:
```suggestion
    "description": "Known as \"a worse version of petrification bracelet\", this tool casts a spell that temporarily breaks the whole movement system of creature, disabling its movement for a short time.  Require almost a week to restore its charge.",
```

File: data/mods/Magiclysm/items/enchanted_bracelet.json [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "holster": true, "ammo_restriction": { "crystallized_mana": 2 } } ],
+    "relic_data": { "charge_info": { "recharge_type": "periodic", "time": "7 d", "regenerate_ammo": true } },
+    "use_action": { "type": "cast_spell", "spell_id": "disorient_spell", "no_fail": true, "level": 0, "need_worn": true }
+  },
+  {
+    "copy-from": "platinum_bracelet_magic",
+    "id": "super_hot_bracelet",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "propulsion bracelet" },
+    "description": "It feels like the world aroud you just stop.  Thats not true, thats just a combination of really fast speed and some mana tricks you can't understand.  Require a whole week to restore his charge.",
```
Comment (TheShadowFerret) [inline data/mods/Magiclysm/items/enchanted_bracelet.json:90]:
```suggestion
    "description": "It feels like the world around you just stops.  That's not true, that's just a combination of really fast speed and some mana tricks you can't understand.  Require a whole week to restore its charge.",
```

**Review by TheShadowFerret — CHANGES_REQUESTED** [review]

File: data/mods/Magiclysm/items/enchanted_bracelet.json (hunk 4 above)
Comment (GuardianDll) [reply data/mods/Magiclysm/items/enchanted_bracelet.json:90]: thank you!

File: data/mods/Magiclysm/Spells/item_only.json [hunk 5]
```diff
@@ -282,5 +282,82 @@
...
+    "min_range": 0,
+    "max_range": 0,
+    "extra_effects": [ { "id": "mana_bolt_random" } ],
+    "energy_source": "MANA"
+  },
+  {
+    "id": "gorgon_spell",
+    "type": "SPELL",
+    "name": "Petrification",
+    "description": "Freeze!.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/item_only.json:321]:
```suggestion
    "description": "Freeze!",
```

**Review by KorGgenT — APPROVED** [review]

**Pull request merged**